### ISSUE TITLE
[duplicate] UI incompatible with new bot version

### ISSUE THREAD

**Issue by icoder-new** [go-telegram/ui#3]
```
# github.com/go-telegram/ui/datepicker
..\..\..\go\pkg\mod\github.com\go-telegram\ui@v0.2.0\datepicker\callback.go:47:31: cannot use update.CallbackQuery.Message (variable of type models.MaybeInaccessibleMessage) as models.InaccessibleMessage value in argument to datePicker.showMain
..\..\..\go\pkg\mod\github.com\go-telegram\ui@v0.2.0\datepicker\callback.go:50:31: cannot use update.CallbackQuery.Message (variable of type models.MaybeInaccessibleMessage) as models.InaccessibleMessage value in argument to datePicker.showMain
..\..\..\go\pkg\mod\github.com\go-telegram\ui@v0.2.0\datepicker\callback.go:54:45: update.CallbackQuery.Message.Chat undefined (type models.MaybeInaccessibleMessage has no field or method Chat)
..\..\..\go\pkg\mod\github.com\go-telegram\ui@v0.2.0\datepicker\callback.go:55:45: update.CallbackQuery.Message.MessageID undefined (type models.MaybeInaccessibleMessage has no field or method MessageID)
..\..\..\go\pkg\mod\github.com\go-telegram\ui@v0.2.0\datepicker\callback.go:62:31: cannot use update.CallbackQuery.Message (variable of type models.MaybeInaccessibleMessage) as models.InaccessibleMessage value in argument to datePicker.onSelect
..\..\..\go\pkg\mod\github.com\go-telegram\ui@v0.2.0\datepicker\callback.go:66:45: update.CallbackQuery.Message.Chat undefined (type models.MaybeInaccessibleMessage has no field or method Chat)
..\..\..\go\pkg\mod\github.com\go-telegram\ui@v0.2.0\datepicker\callback.go:67:45: update.CallbackQuery.Message.MessageID undefined (type models.MaybeInaccessibleMessage has no field or method MessageID)
..\..\..\go\pkg\mod\github.com\go-telegram\ui@v0.2.0\datepicker\callback.go:74:31: cannot use update.CallbackQuery.Message (variable of type models.MaybeInaccessibleMessage) as models.InaccessibleMessage value in argument to datePicker.onCancel
..\..\..\go\pkg\mod\github.com\go-telegram\ui@v0.2.0\datepicker\callback.go:76:37: cannot use update.CallbackQuery.Message (variable of type models.MaybeInaccessibleMessage) as models.InaccessibleMessage value in argument to datePicker.showSelectYear
..\..\..\go\pkg\mod\github.com\go-telegram\ui@v0.2.0\datepicker\callback.go:78:37: cannot use update.CallbackQuery.Message (variable of type models.MaybeInaccessibleMessage) as models.InaccessibleMessage value in argument to datePicker.showSelectYear
..\..\..\go\pkg\mod\github.com\go-telegram\ui@v0.2.0\datepicker\callback.go:78:37: too many errors
```

Getting error While I'm using go-telegram/bot verion 1.1.4 and go-telegram/ui version 0.2.0! 
To solve problem I used 1.0.1 version of go-telegram/bot!

P.S: It is similar to error in issue #1

**Comment by microgiantya** [comment]
Same thing
```
# github.com/go-telegram/ui/keyboard/inline
/home/xxx/go/pkg/mod/github.com/go-telegram/ui@v0.2.0/keyboard/inline/callback.go:31:44: update.CallbackQuery.Message.Chat undefined (type models.MaybeInaccessibleMessage has no field or method Chat)
/home/xxx/go/pkg/mod/github.com/go-telegram/ui@v0.2.0/keyboard/inline/callback.go:32:44: update.CallbackQuery.Message.MessageID undefined (type models.MaybeInaccessibleMessage has no field or method MessageID)
/home/xxx/go/pkg/mod/github.com/go-telegram/ui@v0.2.0/keyboard/inline/callback.go:53:39: cannot use update.CallbackQuery.Message (variable of type models.MaybeInaccessibleMessage) as models.InaccessibleMessage value in argument to kb.handlers[btnNum].Handler
```
So sad by that ((

**Comment by negasus** [comment]
Released v0.3.0
